datasetinsights/io/downloader/http_downloader.py

```diff
@@ -51,3 +62,74 @@ def download(self, source_uri, output, checksum_file=None, **kwargs):
                 logger.info("Checksum mismatch. Deleting the downloaded file.")
                 os.remove(dataset_path)
                 raise e
+
+
+def download_dataset_from_http_url(source_uri, output):
```

Comment:
Can this method reuse `datasetinsights.io.download.download_file`? This method seems to be a duplicate.

Also, you should make this private static method for `HTTPDatasetDownloader`.